gulpfile.js

```diff
@@ -67,7 +67,7 @@ gulp.task(
               }))
               .pipe(tslint.report()));
 
-gulp.task('depcheck', () => {
+gulp.task('depcheck', ['build'], () => {
```

Comment:
why build first? it should be analyzing our typescript directly

Reply:
Without this change it crashes saying that `chokidar` is unused.

Adding some console.log statements around makes me think that it's unable to handle `lint/lint.ts`, giving a warning in the babylon parser, a little bit after an `await` statement. It seems like it's not handling typescript perfectly. Fortunately by ensuring that `build` happens first, it will also run over our generated JS, which it does handle correctly.